contrib/scripts.py

```diff
@@ -9,15 +9,16 @@
 from typing import TYPE_CHECKING
 
 import requests
-from beancount.query import query_env
-from beancount.query import query_parser
+from beanquery import query_compile
+from beanquery.parser.parser import KEYWORDS
+from beanquery.sources.beancount import TABLES
```

Comment:
`beanquery.sources.beancount.TABLES` is an implementation details. The less likely to break in way to introspect the available tables is to look at the `TABLES` member of a properly initialized `beanquery.Connection` object:
```
conn = beanquery.connect('beancount:', ...)
conn.TABLES
```

Reply:
as written above, probably doesn't matter too much in this file.

Anyway, tried with conn.TABLES and that throws an AttributeError.

```
conn = connect(
    "beancount:", entries=[], options=OPTIONS_DEFAULTS, errors=[]
)
print(conn.TABLES)
```

throws `AttributeError: 'Connection' object has no attribute 'TABLES'`

Reply:
Sorry, I was typing from memory. The attribute name is lowercase `conn.tables`

Reply:
thanks, yeah, that works now :)